File: src/main/java/ru/citeck/ecos/completion/java/JavaCompletionContributor.java
package ru.citeck.ecos.completion.java;

import com.intellij.lang.Language;
import com.intellij.lang.java.JavaLanguage;
import ru.citeck.ecos.completion.CompletionContributor;

public class JavaCompletionContributor extends CompletionContributor {
    @Override
    public Language getLanguage() {
        return JavaLanguage.INSTANCE;
    }
}

File: src/main/kotlin/ru/citeck/completetion/java/JavaCompletionContributor.kt
package ru.citeck.completetion.java

import com.intellij.codeInsight.completion.*
import com.intellij.patterns.PlatformPatterns


class JavaCompletionContributor : CompletionContributor() {

    init {
        extend(CompletionType.BASIC, PlatformPatterns.psiElement(), QNamesCompletionProvider())
    }
}